getting_started/first_3d_game/03.player_movement_code.rst

```diff
@@ -12,6 +10,8 @@ Right-click the ``Player`` node and select *Attach Script* to add a new script t
 it. In the popup, set the *Template* to *Empty* before pressing the *Create*
 button.
 
+We set it to *Empty* because we want to write our own code for player movement.
```

Comment:
```suggestion
it. In the popup, set the *Template* to *Empty* before pressing the *Create*
button. We set it to *Empty* because we want to write our own code for
player movement.
```